expo-maps/src/Map.tsx

```diff
@@ -23,6 +23,12 @@ const defaultNativeExpoMapViewProps: DefaultNativeExpoMapViewProps = {
   enableScrollGestures: true,
   enableTiltGestures: false,
   enableZoomGestures: true,
+  cameraPosition: {
+    latitude: 51.51,
+    longitude: 0.13,
+    zoom: 4,
+    animate: true,
+  },
```

Comment:
What should be the default value here? Currently it is London (randomly picker it)

Reply:
London is fine I think